frontend/src/pulldasher/pulls-context.tsx

```diff
@@ -0,0 +1,33 @@
+import { createContext, useContext } from 'react';
+import { useFilteredPullsState, FilterFunction, FilterFunctionSetter } from './filtered-pulls-state';
+import { Pull } from '../pull';
+
+interface PullContextProps {
+   // Array of pulls passing the filter function
+   pulls: Pull[];
+   // Changes the filter function
+   setFilter: FilterFunctionSetter;
+}
+
+const defaultProps = {
+   pulls: [],
+   // Default implementation is a no-op, just so there's
+   // something there until the provider is used
+   setFilter: (filter: FilterFunction) => filter,
```

Comment:
Might want
```suggestion
   setFilter: (filter: FilterFunction) => undefined,
```
because `FilterFunctionSetter` has a `void` return type.

Reply:
That was my first inclination too, but then `eslint` complained that I had an unused variable (the `filter` arg). I figured this was better than an `eslint-disable ...`